lib/matplotlib/collections.py

```diff
@@ -2201,6 +2201,7 @@ def get_cursor_data(self, event):
         contained, info = self.contains(event)
         if len(info["ind"]) == 1:
             ind, = info["ind"]
-            return self.get_array()[ind]
+            array = self.get_array()
+            return self.get_array()[ind] if array else None
```

Comment:
Should use `array` instead of calling `get_array` again.

Reply:
yep sure! don't know why I didn't do that in the first place...